app/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,11 @@
  @import "bootstrap-sprockets";
  @import "bootstrap";
+ @import "users.scss";
+
+body{
+  padding-top: 60px;
+  background-color:#E8E8E8;
+  padding-bottom: 30px;
+  padding-left: 30px;
+  font: helvetica;
```

Comment:
```css
body {
  padding: 60px 30px;
  background: #e8e8e8;
  font-family: Helvetica, Arial, sans-serif;
}
```
So, some general rules for CSS: 
- When you have lots of 'padding' or 'margin' rules, either group them all together or, preferably, use the [shorthand](http://www.w3schools.com/css/css_padding.asp) of putting them all together into one rule. 
- Always capitalize your fonts, and add 'fallback' fonts including generic 'serif' or 'sans-serif' in case the user's computer doesn't have your first choice installed.
- When ordering your css properties, you should follow the order of `box, border, background, text, other`. Box is anything to do with positioning, margins, padding. The others are fairly self-explanatory. 
- Spacing consistency is important. Generally you want a space after the rule declaration and before the squiggly bracket (`body {`). You also want a space after `:`